.travis.yml

```diff
@@ -40,6 +40,10 @@ matrix:
       dist: xenial
       env:
         - TOX_ENV=py38
+    - python: 3.9
```

Comment:
```suggestion
    - python: 3.9dev
```

Reply:
We can use `3.9-dev`, but I think I'd rather just wait a few days for Travis to update to the stable version of 3.9.